Tests/HelloWorld/tactility.py

```
@@ -0,0 +1,693 @@
+    if elf_path is not None:
+        os.remove(elf_path)
+    if skip_build:
+        return True
+    print(f"Building first {platform} build")
+    cmake_path = get_cmake_path(platform)
+    print_status_busy(f"Building {platform} ELF")
+    shell_needed = sys.platform == "win32"
+    build_command = ["idf.py", "-B", cmake_path, "build"]
+    if verbose:
+        print(f"Running command: {" ".join(build_command)}")
+    with subprocess.Popen(build_command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=shell_needed) as process:
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)